rust/noosphere-ns/src/dht/channel.rs

```diff
@@ -91,6 +92,16 @@ impl<Q, S, E> MessageClient<Q, S, E> {
     }
 }
 
+// Manually implement `Clone` so that the generics do not need
+// also implement.
+impl<Q, S, E> Clone for MessageClient<Q, S, E> {
```

Comment:
Why was this needed? Is the new storage not safely `Clone`?

Reply:
IIRC the `anyhow::Error` (`E`) containing the Sqlite error could not be cloned, this could be reworked however